pyads/pyads_ex.py

```diff
@@ -750,7 +947,7 @@ def adsSyncReadByNameEx(
     port,
     address,
     data_name,
-    data_type,
+    data_type=None,
```

Comment:
Do I understand it right that you read the datatype from the PLC if None is passed here? Nice feature.

Reply:
That's right, but now I'm not sure this should be included in this PR...
This change is actually proposed in #74 - I brought the code from that PR into this one because I needed to use the SAdsSymbolInfo code that was introduced there. But this PR does not actually use this function.
So I think I should either:
a) exclude this change from this PR, or;
b) expand the scope of this PR to include the functionality proposed in #74 (I have already done the symbol info caching in this PR, for example)
What do you think?

Reply:
As the author of that PR, glad to see even part of it being put to good use. :+1:

Reply:
For the sake of tracability it would be better to split these two features in two PRs. However since you already included it in the current PR I would opt for b) to save you the hassle. Just make sure to add a description to the Changelog.